### PR TITLE
ISSITES-6093 update info/widget templates and add specification template

### PR DESCRIPTION
**Pull request by jotchon**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-vocabulary-facets/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.vocab-facet .panel-body {
+	padding: 0.5rem 1rem 1rem;
+}
+
+.vocab-facet .list-unstyled {
+	margin-bottom: 0;
+}
+</style>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-vocabulary-facets/ddm-template.ftl:22]: Could you please format these styles? Brian always asks us to do this.
```suggestion
<style>
   .vocab-facet {
 	   border-radius: 10px;
   }

   .vocab-facet .panel a {
	   padding: 1rem;
   }

   .vocab-facet .collapse-icon .collapse-icon-closed .lexicon-icon,
   .vocab-facet .collapse-icon .collapse-icon-open .lexicon-icon {
	   margin-top: 0.3rem;
   }

   .vocab-facet .panel-body {
	   padding: 0.5rem 1rem 1rem;
   }

   .vocab-facet .list-unstyled {
	   margin-bottom: 0;
   }
</style>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/price-model-specification-facet/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+.price-model-facet .panel-body {
+	padding: 0.5rem 1rem 1rem;
+}
+
+.price-model-facet .list-unstyled {
+	margin-bottom: 0;
+}
+</style>
+<@liferay_ui["panel-container"]
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/price-model-specification-facet/ddm-template.ftl:23]: Could you please format these styles? Brian always asks us to do this.
```suggestion
<style>
   .price-model-facet {
	   border-radius: 10px;
   }

   .price-model-facet .panel a {
	   padding: 1rem;
   }

   .price-model-facet .collapse-icon .collapse-icon-closed .lexicon-icon,
   .price-model-facet .collapse-icon .collapse-icon-open .lexicon-icon {
	   margin-top: 0.3rem;
   }

   .price-model-facet .panel-body {
	   padding: 0.5rem 1rem 1rem;
   }

   .price-model-facet .list-unstyled {
	   margin-bottom: 0;
   }
</style>

<@liferay_ui["panel-container"]
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 3]
```diff
@@ -1,87 +1,98 @@
 <style type="text/css">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:1]: Could you please format this file?

Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:1]: Could you please format this file?

**Review by silasbsi — CHANGES_REQUESTED** [review]
Good job, but I added some suggestions.

**Comment by jotchon** [comment]
Opened a PR at [#4329](https://github.com/pt-liferay-solutions/liferay-portal/pull/4329)

**Pull request closed without merging**